### ISSUE TITLE
Information on Component, Module, FunctionalComponent is invisible

### ISSUE THREAD

**Issue by jakebeal** [SynBioHub/synbiohub#644]
In the SynBioHub display, a Component, Module, or FunctionalComponent links directly to its definition, and nowhere is there display of information that is actually located on such an item, such as its role or custom annotations.

**Comment by cjmyers** [comment]
That is correct.  We are open to suggestions on how to display this.  There is a balancing act between being complete and overloading users with too much information.

**Comment by jakebeal** [comment]
How about, in the case where there is interesting additional information, there is a little "expand sub-table" widget that can open up the properties right there at the reference?

**Comment by cjmyers** [comment]
Seems reasonable though may not be done for a bit.  Is there any part of that information which is a priority to get up with a simpler UI change?

**Comment by jakebeal** [comment]
The information that I'd most like to see exposed is annotations --- in particular, the concentration information that we add to FunctionalComponent when defining media

**Comment by cjmyers** [comment]
BTW: if you enter the URI for the FunctionalComponent, you should get a page for the FunctionalComponent.  Indeed, all URIs in SBOL resolve to pages, they are just not all linked in directly.

**Comment by cjmyers** [comment]
Done in PR #757
